lib/tasks/cap_cutover.rake

```diff
@@ -102,363 +33,13 @@ namespace :cap_cutover do
-    @cap_manual_import_logger.info 'Started cap manual pub import process ' + Time.zone.now.to_s
-    lines = CSV.foreach(args.file_location, headers: true, header_converters: :symbol) do |row|
-      begin
-        total_running_count += 1
-        Rails.logger.debug "#{total_running_count} in #{distance_of_time_in_words_to_now(start_time, true)}" if total_running_count % 500 == 0
-        author = Author.where(sunetid: row[:sunetid]).first
-        if author.nil?
-          @cap_manual_import_logger.warn "Unable to find: #{row[:sunetid]}. skipping #{row[:article_title]}"
-          next
-        end
-        pub_hash = convert_manual_publication_row_to_hash(row, author.id.to_s)
-        original_source = row.to_s
-        result = author.publications.where(title: row[:article_title])
-        if result.count > 1
-          @cap_manual_import_logger.warn "Article to update ambiguous for: #{row[:sunetid]}. skipping '#{row[:article_title]}'"
-          next
-        end
-        if (result.count == 1)
-          pub = result.first
-          man_record = UserSubmittedSourceRecord.where(publication_id: pub.id).first
-          if man_record.nil?
-            # Publication exists but UserSubmittedSourceRecord does not
-            @cap_manual_import_logger.info "Publication '#{row[:article_title]}' exists but SourceRecord does not for #{row[:sunetid]}"
-            pub.user_submitted_source_records.create(
-              is_active: true,
-              source_fingerprint: Digest::SHA2.hexdigest(original_source),
-              source_data: original_source,
-              title: pub_hash[:title],
-              year: pub_hash[:year]
-            )
-            pub.send(:update_any_new_contribution_info_in_pub_hash_to_db)
-            pub.sync_publication_hash_and_db
-            pub.save
-          else
-            # Pub and UserSubmittedSourceRecord exist
-            @cap_manual_import_logger.info "Updating Publication '#{row[:article_title]}' for #{row[:sunetid]}"
-            pub.update_manual_pub_from_pub_hash(pub_hash, original_source)
-          end
-        else
-          # Brand new UserSubmittedSourceRecord
-          @cap_manual_import_logger.info "Creating brand new Publication '#{row[:article_title]}'  and SourceRecord for #{row[:sunetid]}"
-          pub = Publication.build_new_manual_publication(pub_hash, original_source)
-          pub.save
-        end
-      rescue => e
-        @cap_manual_import_logger.error "Problem with #{row.inspect}"
-        @cap_manual_import_logger.error e.inspect << "\n" << e.backtrace.join("\n")
-      end
-    end
-    @cap_manual_import_logger.info 'Finished import.' + Time.zone.now.to_s
-    @cap_manual_import_logger.info lines.to_s + ' lines of file: ' + args.file_location.to_s + ' were processed.'
-    @cap_manual_import_logger.info total_running_count.to_s + 'records were processed in ' + distance_of_time_in_words_to_now(start_time, true)
-  end
```

Comment:
OK, `man_pubs_from_delta` noted for deprecation in https://github.com/sul-dlss/sul_pub/issues/356#issuecomment-341251365